Xamarin.Forms.Core/Xamarin.Forms.Core.csproj

```diff
@@ -20,6 +20,9 @@
 		<PackageReference Include="System.Dynamic.Runtime" Version="4.3.0" />
 		<PackageReference Include="System.ValueTuple" Version="4.5.0" />
 	</ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="GitInfo" Version="2.0.20" PrivateAssets="All" />
```

Comment:
Typically, this goes into a `Directory.Build.props/targets` instead

Reply:
👍